CHANGELOG.md

```diff
@@ -7,10 +7,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v1.0.0-rc.1] - 2023-09-06
+
```

Comment:
```suggestion
```
Let's keep the release separate. In any case we cannot release a version 1.0 while we have 0.x dependencies.

Reply:
Yes, please.